.github/labels.md

```diff
@@ -0,0 +1,234 @@
+
+#### Label `status:cherry-pick for release`
+
+The Pull Request should be applied to the currently open release branch.  Use
+this label sparsely, only for bug fixes, and document a minimal set of commits
+that should be considered by the release manager for cherry-picking.
+Cherry-picking Pull Requests onto release branches should fix bugs, not
+introduce new ones.
+
+#### Label `status:duplicate`
+
+The topic of the Issue is or was already adressed by another Issue.  Add the
+number of the adressing Issue in a comment and close this Issue.  If the
+closed Issue contains new knowledge that is valuable to solve the adressing
+Issue copy it over to the adressing Issue.
+
+#### Label `status:follow up`
+
+The Pull Request discussion showed that the proposed implementation is
+insufficient in some way or the Pull Request in itself is malformed.  Usually
+not required as we prefer rebasing within the same Pull request to update
+the implementation.  When applying this Label reference the new Pull Request
+number in the old one.
+
+#### Label `status:help wanted`
+
+??? I don't think we really need that.  At least I didn't see any Issue were it
+caused someone required to help to magically appear.  This can be handled better
+with `@`-mentions if someones or some groups special knowledge is required.
+
+#### Label `status:invalid`
+
+The Issue contains content that fails to adress it intention.  An example
+would be a feature request wasn't discussed before in the forums or for Pull
+Requests those requests that compete to adress the same topic, but were not
+chosed to be merged.  Those Issues won't be merged.
+
+#### Label `status:merged`
+
+All relevant commits of this Pull Request were merged into the master
+development branch.  This can either happen by merging, cherry-picking or
+rebasing and pushing onto master.  Close the Issue if GitHub didn't closed
+it automatically.
+
+#### Label `status:resolved`
+
+The Issue was solved satisfying, either by answering properly or fixing the
+underlying bug.  Close the Issue.
+
+#### Label `status:testing requested`
+
+The Implementation is ready for testing but can't be tested sufficiently by the
+developer and he request testing by other developers.  Use this in case of
+platform specific code or bug fixing when the Bug is not reproducable on the
+developer machine.
+
+#### Label `status:won't fix`
+
+The percieved bug described in the Issue is actually not a bug but intended
+behaviour.  Add a comment explaining why it is intended behaviour and close the
+Issue.  This is __NOT__ a way to brush away user concerns or bad user
+experience so rethink if the Issue can be avoided by redesigning the application
+part in question.
+
+#### Label `status:workaround`
+
+??? I don't know if we really need this.  I intended it for noting a potential
+available workaround for a Issue but does it matter that much that we can't
+write a comment for it and instead need a label?
+
```

Comment:
I understand this label to be primarily for PRs that provide a fix/solution which is either temporary (because a fix is needed but not enough time to implement a thorough fix) or addresses an issue/bug which (for whatever reason) can't be fixed properly, but only by a typical "workaround".

Using this labels for such cases makes it easy to query for such "workaround" fixes to follow up with more thorough/proper fixes later on.

Will probably remain a rare exception, but nonetheless useful to have.